runtime/manual/references/cheatsheet.md

```diff
@@ -8,7 +8,7 @@ title: "Node → Deno cheatsheet"
 | `ts-node file.ts`                      | `deno run file.ts`                             |
```

Comment:
This table can be further simplified. here's my version - please copy


<img width="412" alt="Screenshot 2024-08-07 at 4 24 37 PM" src="https://github.com/user-attachments/assets/52b00499-9b77-4062-8f0f-555854704949">